transformations/GermanGenderSwap/transformation.py

```diff
@@ -0,0 +1,94 @@
+
+
+
+import random
+
+from typing import List
+from checklist.editor import Editor
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+def german_nouns(text, german_pairs):
+    for name in german_pairs.keys():
+        if name in text:
+            text = text.replace(name, german_pairs[name])
+    return text 
+german_pairs = {
```

Comment:
This list looks somewhat limited. Could you add more Nouns?

Reply:
@sebastianGehrmann 
I've updated the transformation with 186 nouns pairs, 2000 male and female first names and also considered some cases of pronouns.
Thanks for you time for the review.
Looking forward to any other comments.